tests/pools/cluster/dynamic.test.js

```diff
@@ -87,4 +86,19 @@ describe('Dynamic cluster pool test suite ', () => {
     const res = await pool1.execute({ test: 'test' })
     expect(res).toBeFalsy()
   })
+  it('Verify scale processes up and down is working when long running task is used', async () => {
```

Comment:
Could you add blank lines between an `it` call and the other `it` call above?

Reply:
Done